lib/converter/rule.go

```diff
@@ -73,6 +73,17 @@ func ruleAPIToBackend(ar api.Rule) model.Rule {
 		})
 	}
 
+	var srcSANames, srcSASelector, dstSANames, dstSASelector string
+	if ar.Source.ServiceAccounts != nil {
```

Comment:
@robbrockbank could u please confirm if I remove SrcSANames and SrcSASelectors then we will not lose any state when going from API to backend. I am guessing the logic is v3->v1 (and now everything is in the selector) and then to backend.